### ISSUE TITLE
gmail sync problems

### ISSUE THREAD

**Issue by Valentyn-Kubrak** [orocrm/crm#93]
Sync with gmail not working properly.
stack trace in oro:cron:imap-sync job:

```
Loaded 0 email address(es).
The synchronization failed. Error: DateTime::__construct(): Failed to parse time string (Fri, 19 Jul 2013 07:27:24 +0800 (GMT+08:00)) at position 36 (+): Double timezone specification.
The synchronization failed. Error: DateTime::__construct(): Failed to parse time string (Fri, 19 Jul 2013 07:27:24 +0800 (GMT+08:00)) at position 36 (+): Double timezone specification.
Finding an email origin ...
```

**Comment by alsma-magecore** [comment]
Hey @Valentyn-Kubrak ! Could you please check this one more time ? We did a fix in 1.4.2 version, so now it should work properly.

**Comment by artml** [comment]
No activity on this issue for more than a year so I'm closing it.
